eng/helix/content/runtests.cmd

```diff
@@ -22,6 +22,8 @@ set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 set DOTNET_MULTILEVEL_LOOKUP=0
 set DOTNET_CLI_HOME=%HELIX_CORRELATION_PAYLOAD%\home
 
+echo Original path %PATH%
+setlocal
```

Comment:
Doesn't https://github.com/dotnet/aspnetcore/pull/23577/files#diff-28e9118e047c42bc3f4f81273511de75R3 already enter a local block? And, even if it doesn't why would the various environment settings in this script persist -- was it just due to the missing `endlocal` (though there's supposed to be an implied `endlocal` at the end of the script)❔ 

Is it possible the previous `setlocal` is failing because Windows on ARM doesn't support `ENABLEDELAYEDEXPANSION`❔

Reply:
I don't think this is the fix. The problem very likely stems from running dotnet-install.ps1 without the -NoPath argument. See https://github.com/dotnet/aspnetcore-internal/issues/3628#issuecomment-652650870 for a summary.